docs/installation.rst

```diff
@@ -66,6 +66,12 @@ Execute the following command to install the latest development build of Transfo
 
 This will automatically detect if any supported deep learning frameworks are installed and build Transformer Engine support for them. To explicitly specify frameworks, set the environment variable `NVTE_FRAMEWORK` to a comma-separated list (e.g. `NVTE_FRAMEWORK=jax,tensorflow`).
 
+.. code-block:: bash
+
+  pip install git+https://github.com/NVIDIA/TransformerEngine.git@refs/pull/NNN/merge
+
+To install a specific PR. Replace NNN by the PR number.
```

Comment:
I think it would be better to start with something like "In order to install a specific PR, execute" and then proceed with the code snippet rather than start with the code snippet without introduction.

Reply:
I amended the commit.
I did that to follow what was already there. But I prefer as you.